0.4.1: windows build failure

I [upgraded](https://github.com/matthiaskrgr/cargo-cache/commit/b3d11d3e6d783081fc1be2881445e2c5645c0da4) `home` to `0.4.1` in `cargo-cache` https://github.com/matthiaskrgr/cargo-cache which caused a CI error / build failure on windows because `home` failed to build:
````
   Compiling home v0.4.1
error[E0308]: mismatched types
  --> C:\Users\travis\.cargo\registry\src\github.com-1ecc6299db9ec823\home-0.4.1\src\lib.rs:89:37
   |
89 |                 let _ = CloseHandle(*h);
   |                                     ^^ expected *-ptr, found enum `std::ffi::c_void`
   |
   = note: expected type `*mut std::ffi::c_void`
              found type `std::ffi::c_void`
````

https://travis-ci.org/matthiaskrgr/cargo-cache/jobs/568579013#L175

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -86,7 +86,7 @@ fn home_dir_() -> Option<PathBuf> {
                 return None;
             }
             let _g = scopeguard::guard(token, |h| {
-                let _ = CloseHandle(*h);
+                let _ = CloseHandle(h);
             });
             fill_utf16_buf(
                 |buf, mut sz| {

